src/scripts/main.js

```diff
@@ -354,7 +354,35 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach((person) => {
+  const lineContainer = document.createElement('tr');
+  const lineName = document.createElement('th');
+  const lineGender = document.createElement('th');
+  const lineBorn = document.createElement('th');
+  const lineDied = document.createElement('th');
+  const lineAge = document.createElement('th');
+  const lineCentery = document.createElement('th');
```

Comment:
Using 'th' elements for non-header cells is semantically incorrect. You should use 'td' elements for table data cells instead.